autobot-backend/mcp/mcp_security_test.py

```
@@ -126,9 +126,27 @@ def admin_client(app):
+    different shards or workers delete each other's directory mid-run. The
+    symptom is an inverted SECURITY assertion rather than a missing-file error
+    -- with the forbidden target deleted, a symlink chain into it dangles, and
+    a dangling link resolves to a non-existent path that is still under the
+    allowed root, so ``is_path_allowed`` returns True and the test reports
+    "symlink chain escape was not blocked". The escape was not the bug; the
+    fixture was.
+
+    ``tmp_path`` supplies the uniqueness because pytest already guarantees it
+    is distinct per test and per worker.
+    """
+    test_dir = Path("/tmp/autobot") / f"test_security_{tmp_path.name}"  # nosec B108  # allowed root, unique leaf
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp/autobot"

(S108)